packages/devtools/src/commands/getElementProperty.ts

```diff
@@ -9,14 +9,22 @@
  */
 
 import { getStaleElementError } from '../utils'
+import type DevToolsDriver from '../devtoolsdriver'
 
-export default async function getElementProperty ({ elementId, name }) {
+export default async function getElementProperty (
+    this: DevToolsDriver,
+    { elementId, name }: { elementId: string, name: string }
+) {
     const elementHandle = await this.elementStore.get(elementId)
 
     if (!elementHandle) {
         throw getStaleElementError(elementId)
     }
 
     const jsHandle = await elementHandle.getProperty(name)
+    if (!jsHandle) {
```

Comment:
Not sure if it's a valid behavior, need to check how it works in webdriver protocol

Reply:
you are right, if property is not found we return `null`:
```
2020-11-13T10:27:36.596Z INFO webdriver: COMMAND findElement("css selector", "body")
2020-11-13T10:27:36.596Z INFO webdriver: [POST] http://localhost:4444/session/e96c697be98948564f5673d379ca9193/element
2020-11-13T10:27:36.596Z INFO webdriver: DATA { using: 'css selector', value: 'body' }
2020-11-13T10:27:36.622Z INFO webdriver: RESULT {
  'element-6066-11e4-a52e-4f735466cecf': '3042905e-7556-4d50-bf88-b822eb830b54'
}
{
  'element-6066-11e4-a52e-4f735466cecf': '3042905e-7556-4d50-bf88-b822eb830b54'
}
› browser.getElementProperty("3042905e-7556-4d50-bf88-b822eb830b54", "foobar")
2020-11-13T10:27:53.246Z INFO webdriver: COMMAND getElementProperty("3042905e-7556-4d50-bf88-b822eb830b54", "foobar")
2020-11-13T10:27:53.247Z INFO webdriver: [GET] http://localhost:4444/session/e96c697be98948564f5673d379ca9193/element/3042905e-7556-4d50-bf88-b822eb830b54/property/foobar
null
```